src/components/StatusUpdater/index.tsx

```diff
@@ -7,25 +7,43 @@ import { UserInfo } from '@respond/types/userInfo';
 import { SplitButton } from '../SplitButton';
 import { useFormLogic, UpdateStatusForm } from './UpdateStatusForm';
 
-const options = [
-  { id: ResponderStatus.Unavailable, text: 'Not Available' },
-  { id: ResponderStatus.Standby, text: 'Stand By' },
-  { id: ResponderStatus.SignedIn, text: 'Sign In' },
-  { id: ResponderStatus.SignedOut, text: 'Sign Out' },
-]
-const optionTexts = options.reduce((accum, cur) => ({ ...accum, [cur.id]: cur.text }), {} as Record<string, string>);
+const options = {
+  standby: { id: ResponderStatus.Standby, text: 'Stand By' },
+  signin: { id: ResponderStatus.SignedIn, text: 'Sign In' },
+  signout: { id: ResponderStatus.SignedOut, text: 'Sign Out' },
+}
+const optionTexts = Object.values(options).reduce((accum, cur) => ({ ...accum, [cur.id]: cur.text }), {} as Record<string, string>);
+
+function isFuture(time: number) {
+  return (time - 60 * 60 * 1000) > new Date().getTime();
+};

```

Comment:
Maybe put 60 * 60 * 1000 into a const so it's self-documenting. I'm guessing it's millisecondsPerHour?